designs/UI/businesses_update.html

```diff
@@ -0,0 +1,127 @@
+                                                <input type="password" class="form-control" id="NAME" placeholder="">
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="form-group">
+                                        <div class="t text-left" style="">BUSINESS LOCATION </div>
+                                        <div class="form-group">
+                                            <div class="inner-addon left-addon">
+                                            <i class="fa fa-building"></i>
+                                                <input type="password" class="form-control" id="NAME" placeholder="">
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="form-group">
+                                        <div class="text-left" style="">BUSINESS EMAIL </div>
+                                        <div class="form-group">
+                                            <div class="inner-addon left-addon">
+                                                <i class="fa fa-envelope"></i>
+                                                <input type="password" class="form-control" id="email" placeholder="">
+                                            </div>
+                                        </div>
+                                    </div>
+                                      
+                                   </form>
+                              </div>
+              
+                              </div>
+                          </div>
+                          <div class="col-xs-6 col-sm-8">
+                              <div class="card">
+                                  <div class="card-header deep-orange lighten-1 white-text">PRODUCTS & SERVICES</div>
+                                  <div class="card-body">
+                                     <h5 class="card-title">Enter more details about the business here</h5>
+                                    <textarea style="width:100%; height:350px;">
+
+                                    </textarea>
+                                    <div class="text-right">
+                                      <span class="text-right"><button class="btn btn-info" >SAVE CHANGES</button></span>
+                                    </div>
+                                    
+                                  </div>
+                              </div>
+
+
+                              <div class="container">
+                              
+                              </div>
+                          <div class="container">
+                          </div>
+                          
+                      </div>
+                  </div>
+</body>
+</html>
```

Comment:
It is nice work done, but try to properly indent your markup(HTML)

Reply:
Ok thank you, I just found an easier way to format in vscode editor